packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1442,4 +1463,74 @@
     <protected>false</protected>
     <shortDescription>Show less</shortDescription>
   </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionTopic</fullName>
+    <value>Is the answer about the right topic?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Question Topic</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionHallucination</fullName>
+    <value>Is the answer free of hallucinated content?</value>
```

Comment:
```suggestion
    <value>Is the answer free of hallucinated content?</value>
```

Is this not too "technical" for users? Seems like a weird way to just say something like: "Is the answer accurate?"
Hallucinations are LLM/Generative AI jargon IMO so I would be tempted to keep it simple. What do you think?

Reply:
Nope the questions have been decided by ML. The target audience is a lot more "Subject matter experts" than general users AFAIK, even if we present it to everyone.
Accurate is already pretty much covered by the other questions too.